docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>Remove "Note that".</p><p> </p><p>Rephrase final sentence to:</p><p> </p><p>"You can list all the experiments you have uploaded by using</p><p> </p><p><code>tensorboard dev list</code></p><p> "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>